docs/docsite/rst/dev_guide.rst

```diff
@@ -4,15 +4,15 @@
 Guidelines for VMware REST module development
 *********************************************
 
-The Ansible VMware REST collection (on `Galaxy <https://galaxy.ansible.com/vmware/vmware_rest>`_, source code `repository <https://github.com/ansible-collections/vmware.vmware_rest>`_) is maintained by Red Hat and the community.
+The Ansible VMware REST collection (on ``Galaxy <https://galaxy.ansible.com/vmware/vmware_rest>``, source code ``repository <https://github.com/ansible-collections/vmware.vmware_rest>``) is maintained by Red Hat and the community.
 
 .. contents::
       :local:
 
 Contribution process
 ====================
 
-The modules of the vmware_rest collection are autogenerated by another tool called `vmware_rest_code_generator <https://github.com/ansible-collections/vmware_rest_code_generator>`.
+The modules of the vmware_rest collection are autogenerated by another tool called ``vmware_rest_code_generator <https://github.com/ansible-collections/vmware_rest_code_generator>``.
```

Comment:
This is wrong, isn't it? I'm pretty sure @bardielle did the last release using [ansible.content_builder](https://github.com/ansible-community/ansible.content_builder).

But I think there are some other places where vmware_rest_code_generator is still used. Should we correct this here, or should we concentrate on fixing the broken doc generation and create another PR for this?

I tend to the second option. What do you think?

Reply:
I think your right, this is out of date.

I agree with your tend-ing. Im supposed to generate specs for vcenter 8 soon so I can document the updated process then